packages/apps/marmalade-marketplace/src/components/Badge/style.css.ts

```diff
@@ -0,0 +1,11 @@
+import { token } from '@kadena/kode-ui/styles';
+import { style } from '@vanilla-extract/css';
+
+export const tokenBadgeWrapperClass = style({
+  position: 'absolute',
+  top: '8px',
```

Comment:
I think these should be styled with `token('spacing.n2')` not sure if that's the correct token though